### ISSUE TITLE
Looking forward

### ISSUE THREAD

**Issue by killthekitten** [andreareginato/betterspecs#66]
I finished translating betterspecs to russian language, and now I am sure, that it will be _very_ hard to support the whole app in future. Here is the list of things that I think will improve the awesomness of betterspecs.org:
- Layout must be DRY and reusable.
- Menu items must be auto-generated from headers instead of being inserted manualy.
- Same with social buttons.
- Same with additional info links.
- Translations should be more maintanable. It will be great if the translation maintainer could receive a letter when some practice was added or changed in original version of the betterspecs.
- Practices should be voted and discussed on the site, not on the github.

In other words, we should start saving data to the DB and write some ruby code to do it more usable. What do you think?

Thanks.
